.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
```

Comment:
"use strict" actually changes the engine's parsing rules, so it might be visual noise, but it has an actual effect in terms of processing speed and error tolerance in the browser.